<?xml version="1.0"?>

<launch>
<!-- 
  <node pkg="tf" type="static_transform_publisher" name="base_link_to_ins" args="0 0 0.5 0 0 0 1 base_link ins 10" />
  <node pkg="tf" type="static_transform_publisher" name="base_link_to_lidar" args="0 0 2.3 0 0 0 1 base_link lidar 10" /> -->

  <node name="fix2pose" pkg="fix2pose" type="fix2pose" output="screen">      
        <!-- <remap from="AA" to="BB"/> -->
      <remap from="/gps" to="/fix"/>
      <remap from="/pose" to="/current_pose"/>

      <!-- UNIST MAP -->
      <!-- <param name="latOrigin" value="35.57083743"/> 
      <param name="lonOrigin" value="129.188112768"/>  
      <param name="altOrigin" value="55.494"/>  -->

      <!-- KIAPI MAP -->
      <param name="latOrigin" value="35.6470671983"/> 
      <param name="lonOrigin" value="128.401913578"/>  
      <param name="altOrigin" value="25.797"/> 



    
  </node>
</launch>
